package.json

```diff
@@ -122,7 +122,9 @@
     "dompurify": "^3.0.8",
     "favico.js-slevomat": "^0.3.11",
     "gifuct-js": "^2.1.2",
+    "hsluv": "^1.0.1",
     "jed": "1.1.1",
+    "js-sha1": "^0.7.0",
```

Comment:
Is this really necessary? Modern browsers support basic crypto primitives and operations.

https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest

Reply:
ok, will fix.